ext/standard/filestat.c

```diff
@@ -42,6 +42,14 @@
 #  include <os2.h>
 #endif
 
+#if defined(__APPLE__)
+// Apple statvfs has an interger overflow in libc copying to statvfs.
+//  cvt_statfs_to_statvfs(struct statfs *from, struct statvfs *to) {
+//  to->f_blocks = (fsblkcnt_t)from->f_blocks;
+#undef HAVE_SYS_STATVFS_H
```

Comment:
I may suggest, for style consistency, to have spaces like above.

Reply:
If I'm following your suggestion correctly, I've made a patch to match styles.

Reply:
8.0.15?

Reply:
PHP-8.0

Reply:
Changed. New PR https://github.com/php/php-src/pull/8056

Reply:
I had not noticed you created from your own `master` branch. it might be better to recreate your patch/PR from PHP-8.0 but with a distinct branch.